Dbal/Configuration.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\Dbal;
```

Comment:
should not be in a `Dbal` namespace as it extends the *ORM* configuration class.

Reply:
moved to a new `ORM` namespace